authors/apps/authentication/serializers.py

```diff
@@ -244,6 +244,23 @@ class ResetPasswordSerializer(serializers.ModelSerializer):
         write_only=True
     )
 
+    def validate_password(self, password):
```

Comment:
Could you reset this function name to
`def validate_reset_password(self, password):`

So that it is easier to follow for clarity. I had initially thought you had changed Edna's validation for the initial password.

Reply:
Hi Jean, Django had a `validate_fieldname()` function that helps in validating a particular field in the serializers. I think he should leave it that way.

Reply:
In reference to the documentation of Django rest framework, to validate a field called "password" the validation method has to be "validate_password".
That name is pre-dictated by the framework